nodes.go

```diff
@@ -225,6 +226,96 @@ func (n *Node) GetMetadata() (string, error) {
 	return md.String(), nil
 }
 
+type updateMetadata struct {
+	Name        *string `json:"name,omitempty"`
+	Description *string `json:"description,omitempty"`
+}
+
+// Updates metadata fields name or description of Node n. If a given field is nil,
+// it will not be updated.
+func (n *Node) UpdateMetadata(name *string, description *string) (*http.Response, error) {
+	metadata := &updateMetadata{
+		Name:        name,
+		Description: description,
+	}
+	url := fmt.Sprintf("nodes/%s", *n.Id)
+	req, err := n.service.client.NewMetadataRequest("PATCH", url, &metadata)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := n.service.client.Do(req, nil)
+	if err != nil {
+		return resp, err
+	}
+	err = resp.Body.Close()
+	return resp, err
+}
+
+type replaceParent struct {
+	FromParent string `json:"fromParent"`
+	ChildId    string `json:"childId"`
+}
+
+func (n *Node) ReplaceParent(oldParentId string, newParentId string) (*http.Response, error) {
```

Comment:
I've checked the implementation - looks good!

Needs a comment as it is an externally visible function.